doc/classes/Curve.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Curve" inherits="Resource" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		A mathematic curve.
+		Describes a mathematical curve.
```

Comment:
Don't think this is change is needed, a mathematical curve isn't a physical thing so saying "A mathematical curve" is sufficient, and just complicates translation

Reply:
I like the new description. The curve itself isn't really stored, only it's parameters that describe it are stored (and potentially a baked abstraction). But either description is fine by me.

Reply:
That is what a curve is though, but either way

Reply:
Good point, I also did this for Gradient.